nav2_costmap_2d/test/unit/declare_parameter_test.cpp

```diff
@@ -34,6 +34,7 @@ class LayerWrapper : public nav2_costmap_2d::Layer
   void updateBounds(double, double, double, double *, double *, double *, double *) {}
   void updateCosts(nav2_costmap_2d::Costmap2D &, int, int, int, int) {}
   bool isClearable() {return false;}
+  bool isPublishable() {return false;}
```

Comment:
Please add the testcase for costmap layers publishers.

Reply:
So, 

this is really only making use of the costmap 2d_publisher which doesn't have any tests written. Also, there was no test written publishing out of the final layer. 

Do you want me to write a for the costmap_2d_publisher? 

I am not really sure what you had in mind for testing the costmay layer publishers.

Reply:
> Also, there was no test written publishing out of the final layer.

Yes, I've checked the codecov of `nav2_costmap_2d` package, and indeed the CostmapPublisher is not being covered by any tests, as well as master layer. In this case, this is not a huge change requiring itself to be covered when testcase exists, but if not - the test coverage of `nav2_costmap_2d` will be another story.